testing/hardhat-deployment/README.md

```
@@ -0,0 +1,46 @@
+- For the wallet, currently only `private_key` is supported.
+
+- For `network`, network name as defined in hardhat.config.ts of the contract repository.
+
+- If the smart contract has prerequisites or dependencies, there are two options:
+    1) If the contracts repository has their dependency in built, provide the full path till the dependency script.
+    2) Else, to set it locally via kurtosis package. Ensure that the `dependency.sh` file inside the `dependency` folder is completed.
+    
+    Set the `dependency` `type` to "git" or "local". This is necessary when additional setup is required before deployment.
+
+## Example for GitHub hosted repository having dependency in the local kurtosis package
+```
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)